src/middleware/authentication_middleware.ts

```diff
@@ -11,5 +11,7 @@ export const authenticationMiddleware = (req: Request, res: Response, next: Next
         returnUrl: BASE_URL + CHECK_SAVED_APPLICATION
     };
 
-    return acspProfileCreateAuthMiddleware(authMiddlewareConfig)(req, res, next);
+    return FEATURE_FLAG_DISABLE_MANDATORY_VERIFICATION === "true"
```

Comment:
May be use isActiveFeature function here.

Reply:
done. Thanks